libsqlite3-sys/upgrade.sh

```diff
@@ -1,28 +1,56 @@
-#!/bin/bash
+#!/bin/bash -e
 
 SCRIPT_DIR=$(cd "$(dirname "$_")" && pwd)
+CUR_DIR=$(pwd -P)
 echo "$SCRIPT_DIR"
-cd "$SCRIPT_DIR" || { echo "fatal error"; exit 1; }
-export SQLITE3_LIB_DIR=$SCRIPT_DIR/sqlite3
+cd "$SCRIPT_DIR" || { echo "fatal error" >&2; exit 1; }
+cargo clean
+mkdir -p "$SCRIPT_DIR/../target" "$SCRIPT_DIR/sqlite3" "$SCRIPT_DIR/sqlcipher"
+export SQLITE3_LIB_DIR="$SCRIPT_DIR/sqlite3"
+export SQLITE3_INCLUDE_DIR="$SQLITE3_LIB_DIR"
+export SQLCIPHER_LIB_DIR="$SCRIPT_DIR/sqlcipher"
```

Comment:
You should add this new directory in `.gitattributes`:
```
libsqlite3-sys/sqlcipher/* linguist-vendored
```

Reply:
I don't yet understand this metadata, but have pushed a commit doing this.

Reply:
Seems this is a good starting point: <https://stackoverflow.com/questions/40659265/using-gitattributes-for-linguist-examples>